Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -0,0 +1,140 @@
+import Foundation
+
+/// A configuration object that defines behavior for code generation.
+public struct ApolloCodegenConfiguration {
+  /// Specify the input files required for code generation.
+  public struct FileInput {
+    /// Path to the GraphQL schema file. Can be in JSON or SDL format.
+    public let schema: URL
+    /// Glob of files to search for GraphQL operations. This should be used to find queries and any client schema extensions. Defaults to `./**/*.graphql`, which will search for `.graphql` files throughout all subfolders of the folder where the script is run.
+    public let glob: String
+    /// [optional] Path to an operation id JSON map file. If specified, also stores the operation IDs (hashes) as properties on operation types. Defaults to `nil`.
+    public let operationIDs: URL?
```

Comment:
I'm pretty sure this is actually an OUTPUT not input. We know how to generate the operationIDs, and we output a file that can be sent to the server to register all your operations. @designatednerd correct me if I'm wrong here.

Reply:
Yes that is definitely an output not an input.

Reply:
Oops, my bad.